src/Assets/LightSkyDefense/Scripts/Controls/FallbackController.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.XR;
+using Valve.VR.InteractionSystem;
+
+public class FallbackController : MonoBehaviour
+{
+    private Player _player;
+    private Hand _fallbackHand;
+
+    private SpawnDialOption[] _dialOptions;
+
+    private GameObject _preview;
+
+    void Start()
+    {
+        _player = Player.instance;
+
+        if (!XRSettings.enabled)
+        {
+            // No HDM found, put options in fallback overlay
+
+            var overlayCanvasText = _player.GetComponentInChildren<Text>();
+            var playerDial = _player.GetComponentInChildren<Dial>(true);
+
+            _dialOptions = playerDial.DialOptions;
+
+            var dailOptionBuilder = new StringBuilder();
+            var dailIndexBuilder = new StringBuilder();
+
+            for (int i = 0; i < playerDial.DialOptions.Length; i++)
+            {
+                dailOptionBuilder.Append("[");
```

Comment:
We should use string interpolation instead of string builder.

```
$"..."
```

Reply:
I moved it to a new method if that makes it clearer.